src/hotspot/share/utilities/vmError.cpp

```diff
@@ -166,6 +170,48 @@ static void print_bug_submit_message(outputStream *out, Thread *thread) {
   out->print_raw_cr("#");
 }
 
+static bool check_stack_headroom(Thread* thread,
```

Comment:
Could you please write a short comment what the return means? From the code, I assume true means "not enough headroom"? Maybe rename function to "stack_has_headroom"?

Reply:
Done.